wagtail/admin/templates/wagtailadmin/shared/status_tag.html

```diff
@@ -0,0 +1,11 @@
+{% if url %}
+    <a href="{{url}}" target="_blank" rel="noreferrer" class="status-tag {% if classname %}{{classname}}{% endif %}" {% if title %}title="{{title}}"{% endif %}>
```

Comment:
```suggestion
    <a href="{{url}}" target="_blank" rel="noreferrer" class='{% classnames "status-tag" classname %}' {% if title %}title="{{title}}"{% endif %}>
```

Using the template util is much easier `classnames`